apps/src/templates/courseOverview/AssignToSection.js

```diff
@@ -75,6 +76,16 @@ class AssignToSection extends Component {
         this.setState({
           sectionIndexToAssign: null
         });
+        firehoseClient.putRecord({
+          study: 'assignment',
+          event: scriptId ? 'unit_overview' : 'course_overview',
```

Comment:
The presence of the `scriptId` currently distinguishes whether the assignment is happening on the unit overview or course overview page. However, if that changes or if we add this component to another page, this wouldn't no longer be accurate. Should I pass in a param for the location of the component instead?

Reply:
This is where the assignment currently happens, but this component won't exist when the new changes go live. This will then need to live in `SectionAssigner` or `AssignButton`, but is ok for now. We can extend to include another param if needed, but those are the two places for now and the foreseeable future.